packages/password/src/accounts-password.ts

```diff
@@ -91,6 +91,10 @@ export interface AccountsPasswordOptions {
    * This function will be called when you call `createUser`.
    */
   validateUsername?(username?: string): boolean;
+  /**
```

Comment:
Since this option is a boolean, can you please move it before the options?
Also, can we add a second line comment with `Default to false.`?

Reply:
I moved it to the other booleans. Sorry for not getting back at this earlier - went on vacation and completely forgot about this as I implemented it manually in our app :/

Wondering if we should default to true though and make this a breaking change.
I guess most users using email/password auth *want* verification mails - afaik in meteor/auth this is the default behavior as well.

Anyhow, we could still do this in a later step.